wallet/api/admin_list_permissions_test.go

```diff
@@ -106,7 +106,7 @@ func testListingPermissionsWithValidParamsSucceeds(t *testing.T) {
 	assert.Equal(t, api.AdminListPermissionsResult{
 		Permissions: map[string]wallet.PermissionsSummary{
 			hostname: {
-				"public_keys": "read",
```

Comment:
Oops ! That shouldn't be renamed.

Reply:
Yes, I will correct that renaming in the wallet. Thanks!

Reply:
Changes in the wallet have been reversed, sorry about that and thanks!